criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language.
+* The codebase MUST be findable using a search engine by codebase name.
+* Maintainers SHOULD submit the codebase to relevant software catalogs.
+* The codebase SHOULD have a website which describes the problem the codebase solves using the preferred jargon of different potential users of the codebase (including technologists, policy experts and managers).
+* The codebase SHOULD have a unique and persistent identifier, such as a [Wikidata item](https://www.wikidata.org/wiki/Wikidata:Introduction) or [FSF software directory listing](https://directory.fsf.org/wiki/Main_Page), mentioning the major contributors, repository location and website.
+* The codebase SHOULD include a machine-readable metadata description, for example in a [publiccode.yml](https://github.com/publiccodeyml/publiccode.yml) file.
+* There MAY be a dedicated domain name for the codebase.
+* The community MAY present regularly at conferences.
+
+## Why this is important
+
+* Proactiveness is needed; just publishing a codebase and hoping it's found doesn't work.
+* The more findable a codebase is, the more potential new collaborators will find it.
```

Comment:
Who should find the codebase? Practically in the case of Open Zaak I don't expect managers to go looking for `publiccode.yaml` on github for example. It seems that word of mouth and the VNG/GEMMA websites are the major drivers for findability.

Reply:
In practice, and considering all types of codebases, I think all roles may go looking for codebases and need to find them. Yes, some codebases may be on the top-of-mind only for certain roles, and some roles are more likely to use a particular tool or method to find the codebase they need. For example, it might only be a handful of people looking for publiccode.yml file, and then it might be another, perhaps slightly larger handful, that looks in the catalogs that scrape these files whereas perhaps the majority will turn to a search engine (unless their country already have a prominent national catalog, like in Italy).